### ISSUE TITLE
Do `gridster` support overlapping of grids?

### ISSUE THREAD

**Issue by vistajess** [swiety85/angular2gridster#180]
Hello , I want to create a web app like a mock up builder where the user can place elements / components wherever he wants.

**Comment by swiety85** [comment]
No, it does not support this case. If you want, I can check what is the effort and than tell you when this feature can be available.
